test/graphtile.cc

```diff
@@ -76,6 +79,11 @@ void id_from_string() {
     GraphTile::GetTileId("foo/bar/1/000/004/001.gph");
     throw std::logic_error("Should fail to get graphtile id");
   } catch(const std::runtime_error&) { }
+
+  try {
+    GraphTile::GetTileId("00/002.gph");
+    throw std::logic_error("Should fail to get graphtile id");
+  } catch(const std::runtime_error&) { }
```

Comment:
if the code were still broken it would return an `out_of_range` exception which would be uncaught and fail the test, but now the code throws a `runtime_error` successfully detecting that this tile path is garbage